docs/wiki/Known issues.md

```diff
@@ -45,9 +45,17 @@ Further details are also provided in issue [#1342](https://github.com/Azure/Reso
 
 ## Microsoft.RecoveryServices/vaults
 
-The Recovery Services Vaults module does not currently validate the identity property (system or user assigned identity).
+The Recovery Services Vaults module does not currently attach the content of the identity property correctly when both user- and systemassigned identity fields are selected.
 
-The module pipeline fails in the deployment validation step when system and user assigned identity parameters are added as input parameters.
+The pipeline shows a success but the assignment of both identities never happens although both identities (systemassigned is a serviceprincipal, userassigned is a managed identity resource) get created successfully.
+
+Upon cleanup the system-assigned identity will not be removed.
+
+When the deployment is then run again it fails, because Azure tries to attach this rogue service principal as a system-assigned identity.
+
+The branch `users/fblix_jpeasier/rsvIdentity` has been used to testing and contains failing and succeeding samples.
+
+Since the behavior is inconsistent via Api (depending on spacing and whether capital letters are used), a ticket on the bicep repository has been opened for that. For more details, refer to the issue in the bicep repository ([#9662](https://github.com/Azure/bicep/issues/9662)).
 
 A related issue has been opened in the Bug board [#2391](https://github.com/Azure/ResourceModules/issues/2391).
```

Comment:
If the above suggestion gets accepted, please remove this paragraph

Reply:
done